tos/templates/tos/tos_check.html

```diff
@@ -1,8 +1,10 @@
 {% load i18n %}
 {% if note %}
-    <h2>{{ note }}</note>
+    <h2>{{ note }}</h2>
 {% else %}
-    <h2>{% blocktrans %}Terms of Service as of {{ tos.created|date:"SHORT_DATE_FORMAT" }}{% endblocktrans %}</h2>
```

Comment:
Was the h2 removed for a reason?

I think blocktranslate rename is only available after django 3.0. Will need to double check. If we roll with that will need to drop CI for django 2. Which is fine, as it's not longer really supported.